<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsGui</class>
 <widget class="QWidget" name="ContactsGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget_contacts">
     <widget class="QWidget" name="page">
      <widget class="QLineEdit" name="lineEdit_age">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>290</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_age">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>290</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Age:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_email">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>230</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_save">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>330</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_lastName">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_firstName">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>140</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_uploadFile">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>330</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload File</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_lastName">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>170</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_phone">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>200</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_email">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>230</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_address">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>260</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_address">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>260</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_phone">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>200</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Phone Number:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_firstName">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
